client/mysqlimport.c

```diff
@@ -431,16 +431,22 @@ static void lock_table(MYSQL *mysql, int tablecount, char **raw_tablename)
 {
   DYNAMIC_STRING query;
   int i;
-  char tablename[FN_REFLEN];
+  char tablename[FN_REFLEN], *pos;
 
   if (verbose)
     fprintf(stdout, "Locking tables for write\n");
   init_dynamic_string(&query, "LOCK TABLES ", 256, 1024);
   for (i=0 ; i < tablecount ; i++)
   {
     fn_format(tablename, raw_tablename[i], "", "", 1 | 2);
-    dynstr_append(&query, tablename);
-    dynstr_append(&query, " WRITE,");
+    dynstr_append(&query, "`");
```

Comment:
@gkodinov thanks. In that case https://github.com/mysql/mysql-server/blob/5.7/client/mysqlimport.c#L387 should be adapted as well.